net.meijn.onvifviewer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "net.meijn.onvifviewer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "onvifviewer",
+    "finish-args": [
+        "--socket=x11",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "kdsoap",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/caspermeijn/KDSoap.git",
+                    "branch": "master",
+                    "commit": "5fadb5eb393ed8aa234290a2001dbe92915907fb"
+                }
+            ]
+        },
+        {
+            "name": "onvifviewer",
+            "buildsystem": "cmake-ninja",
```

Comment:
```json
"config-opts": [
  "-DCMAKE_BUILD_TYPE=Release"
]
```

for both modules.

Reply:
Isn't `DCMAKE_BUILD_TYPE=RelWithDebInfo` better if it's available?

Reply:
Yea, I just always forget it.

Reply:
Should I leave some of the files in /lib/debug as well?